lib/services/platform-environment-requirements.ts

```diff
@@ -136,5 +129,50 @@ export class PlatformEnvironmentRequirements implements IPlatformEnvironmentRequ
 	private fail(message: string): void {
 		this.$errors.fail({ formatStr: message, suppressCommandHelp: true, printOnStdout: true });
 	}
+
+	private getNonInteractiveConsoleMessage(platform: string) {
+		if (this.$nativescriptCloudExtensionService.isInstalled()) {
+			return `${PlatformEnvironmentRequirements.MISSING_LOCAL_SETUP_MESSAGE} To continue, choose one of the following options: ` + EOL
+				+ "Run $ tns setup command to run the setup script to try to automatically configure your environment for local builds." + EOL
+				+ `${this.getCloudBuildsMessage(platform)}` + EOL
+				+ `Verify that your environment is configured according to the system requirements described at ${this.$staticConfig.SYS_REQUIREMENTS_LINK}.`;
+		}
+
+		return `${PlatformEnvironmentRequirements.MISSING_LOCAL_AND_CLOUD_SETUP_MESSAGE}` + EOL
+			+ "Run $ tns setup command to run the setup script to try to automatically configure your environment for local builds." + EOL
+			+ `Run $ tns cloud setup command to install the ${constants.NATIVESCRIPT_CLOUD_EXTENSION_NAME} extension to configure your environment for cloud builds.` + EOL
+			+ `Verify that your environment is configured according to the system requirements described at ${this.$staticConfig.SYS_REQUIREMENTS_LINK}.`;
+	}
+
+	private getInteractiveConsoleMessage(platform: string) {
+		if (this.$nativescriptCloudExtensionService.isInstalled()) {
+			const message = `The ${constants.NATIVESCRIPT_CLOUD_EXTENSION_NAME} extension is installed and you can ${_.lowerFirst(this.getCloudBuildsMessage(platform))}`;
+
+			return `${message.bold}` + EOL
+				+ `${PlatformEnvironmentRequirements.MISSING_LOCAL_SETUP_MESSAGE} To continue, choose one of the following options: ` + EOL
+				+ `Select "Configure for Local Builds" to run the setup script and automatically configure your environment for local builds.` + EOL
+				+ `Select "Skip Step and Configure Manually" to disregard this option and install any required components manually.`;
+		}
+
+		return `${PlatformEnvironmentRequirements.MISSING_LOCAL_AND_CLOUD_SETUP_MESSAGE}` + EOL
+			+ `Select "Configure for Cloud Builds" to install the ${constants.NATIVESCRIPT_CLOUD_EXTENSION_NAME} extension and automatically configure your environment for cloud builds.` + EOL
+			+ `Select "Configure for Local Builds" to run the setup script and automatically configure your environment for local builds.`
+			+ `Select "Configure for Both Local and Cloud Builds" to automatically configure your environment for both options.`
+			+ `Select "Skip Step and Configure Manually" to disregard these options and install any required components manually.`;
```

Comment:
again, some of the messages are duplicated, can you extract them in const variables.